environment/swarmball_env.py

```diff
@@ -15,7 +15,14 @@ def __init__(self, acc_factor=0.25, number_of_clusters=3, **kwargs):
         self.acc_factor = acc_factor
 
     def reward(self):
-        pass
+        tmp = self.goal_prev_pos
```

Comment:
Nie ładniej byłoby to nazwać current_pos i prev_pos zamiast brzydkiego tmp i prev_pos?

Reply:
Zrobiłem jeszcze bardziej intuicyjnym sposobem.

Reply:
Kurde pięknie. Zawsze w Ciebie wierzyłem